### PR TITLE
Intake Subsystem

### LINKED ISSUE #7: Intake subsystem code
# Code for Intake Subsystems
## Mechanisms
- Solenoid for deploy (For coding see [this](https://docs.wpilib.org/en/stable/docs/software/hardware-apis/pneumatics/pneumatics.html#single-solenoids-in-wpilib))
- SparkMax for running motor

### PR DESCRIPTION
**Pull request by David-rel**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import static frc.robot.Constants.IntakeConstants;
+
+
+public class Intake extends SubsystemBase {
+
+  private final CANSparkMax intakeMotor = new CANSparkMax(IntakeConstants.INTAKE_PORT, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+  private final Solenoid exampleSolenoidPCM = new Solenoid(PneumaticsModuleType.REVPH, IntakeConstants.SOLENOID_PORT);
```
Comment (ohowe1) [inline src/main/java/frc/robot/subsystems/Intake.java]: You probably want to change this name

**Review by ohowe1 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  /** Creates a new Intake. */
+  public Intake() {}
+
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+
+  }
+
```
Comment (ohowe1) [inline src/main/java/frc/robot/subsystems/Intake.java]:
```suggestion
```

You don't need any of these empty functions

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  public void setPercentage(double percentage){
+    intakeMotor.set(percentage);   
+  }
+
+  public void setSolinoid(boolean deployed){
```
Comment (ohowe1) [inline src/main/java/frc/robot/subsystems/Intake.java]:
```suggestion
  public void setSolenoid(boolean deployed){
```

**Review by ohowe1 — CHANGES_REQUESTED** [review]
Some small changes

**Review by ohowe1 — APPROVED** [review]

**Comment by ohowe1** [comment]
Awaiting testing

**Review by adambryant — APPROVED** [review]

**Pull request merged**